src/platform/tls_openssl.c

```diff
@@ -32,6 +32,8 @@
 
 uint16_t CxPlatTlsTPHeaderSize = 0;
 
+const size_t OpenSslFilePrefixLength = sizeof("..\\..\\..\\..\\..\\..\\submodules");
```

Comment:
This isn't actually going to need to be a constant in the code, right? If so, I'd MUCH prefer to find another way, maybe to inject it.

Reply:
If you have a better solution, please let me know. This works and gives all the necessary info now:
```
 92 [0]40A4.4CF8::2021/01/29-09:54:48.523204500 [Microsoft-Quic][conn][0x1ED4AFF300] TLS handshake error: error:0A000086:SSL routines::certificate verify failed, file:openssl\ssl\statem\statem_clnt.c:1869
```

Reply:
I'd at least like to a check to make sure we're not going out of bounds if that length changes somehow.